src/ParamConverter/HashidsParamConverter.php

```diff
@@ -4,32 +4,29 @@
 
 namespace Roukmoute\HashidsBundle\ParamConverter;
 
-use Roukmoute\HashidsBundle\Hashids;
+use Hashids\HashidsInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
 use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
 use Symfony\Component\HttpFoundation\Request;
 
 class HashidsParamConverter implements ParamConverterInterface
```

Comment:
would be better to migrate to argument resolvers (a core Symfony feature) rather than the ParamConverter feature of SensioFrameworkExtraBundle.

Reply:
Thanks!

It is currently not possible for the moment:

- https://github.com/doctrine/DoctrineBundle/pull/1413 (currently not merged)
- It is not possible to "passthrough" to another converter, so this means that I would be directly dependent on DoctrineBundle again (like v1), which I would like to avoid if possible.

It will be a v4 :-)